py/server/deephaven/table.py

```diff
@@ -503,10 +505,9 @@ def meta_table(self) -> Table:
     def j_object(self) -> jpy.JType:
         return self.j_table
 
-    def iter_rows(self, cols: Optional[Union[str, Sequence[str]]] = None, *, chunk_size: Optional[int] = 1) \
-            -> Generator[Dict[str, Any], None, None]:
-        """ Returns a generator that reads one row or one chunk of rows at a time from the table into a dictionary.
-        The dictionary is a map of column names to either numpy arrays or scalar values of the column data type.
+    def iter_dict(self, cols: Optional[Union[str, Sequence[str]]] = None) -> Generator[Dict[str, Any], None, None]:
```

Comment:
I think we still need to give user option to specif `chunk_size`.

Reply:
I think it is OK in this case to have a fixed chunk size set internally. The added complication could be confusing and makes it worse for the users.